policyengine_uk/variables/gov/dfe/extended_childcare_entitlement/conditions/extended_childcare_entitlement_work_condition.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_uk.model_api import *
+
+
+class extended_childcare_entitlement_work_condition(Variable):
+    value_type = bool
+    entity = Person
+    label = "Work conditions for extended childcare entitlement"
+    documentation = "Whether the person/couple meets work requirements for extended childcare entitlement"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        benunit = person.benunit
+        is_child = person("is_child", period)
+        in_work = person("in_work", period)
+        is_parent = person("is_parent", period)
+
+        # Get disability status
+        p = parameters(
+            period
+        ).gov.dwp.pension_credit.guarantee_credit.child.disability
+
+        eligible_based_on_disability = (
+            add(
+                person,
+                period,
+                p.eligibility + p.severe.eligibility,
```

Comment:
Replacing the | below
```suggestion
                p.eligibility + p.severe.eligibility + ["incapacity_benefit"]
```

Reply:
On second thought can we make this a list parameter? Where in the statute is this defined?

Reply:
Done.